### ISSUE TITLE
Do you fix the seed when calculating variance?

### ISSUE THREAD

**Issue by ZPBerg** [yaodongyu/Rethink-BiasVariance-Tradeoff#2]
In the paper, you state "We estimate the variance by averaging over N = 3 such random splits (i.e., we train 6 = 3 × 2 copies of each model)." 
So the variance is averaged over 3 different splits of the training data. But to calculate the variance for one of the splits, do you fix the seed, i.e. start from the same set of initial parameters but train on different subsets of the train data?

What do you think is the effect of fixing the seed vs. not fixing the seed?
